modules/core/src/Make.js

```diff
@@ -0,0 +1,139 @@
+import consola from 'consola'
+import prompts from 'prompts'
+import { green, red } from 'kolorist'
+import fs from 'fs-extra'
+import path from 'node:path'
+
+export default class Make {
+  constructor(argv = {}) {
+    this.argv = argv
+  }
+
+  async createModule() {
+    const responses = await this.getResponses()
+    const dst = path.join(process.cwd(), responses.directory, responses.moduleName).replace(/\\/g, '/')
+
+    await fs.mkdir(dst)
+
+    const packageName = responses.vendorName ? `@${responses.vendorName}/${responses.moduleName}` : responses.moduleName
+
+    let packageFile = {
+      name: packageName,
+      version: '0.0.0',
+      license: 'MIT',
+      type: 'module',
+    }
+
+    if (responses.moduleDescription) {
+      packageFile = {
+        ...packageFile,
+        description: responses.moduleDescription,
+      }
+    }
+
+    if (responses.access === 'private') {
+      packageFile = {
+        ...packageFile,
+        private: true,
+      }
+    } else {
+      packageFile = {
+        ...packageFile,
+        publishConfig: {
+          access: responses.access,
+        },
+      }
+    }
+
+    await fs.writeFile(path.join(dst, 'package.json'), JSON.stringify(packageFile))
```

Comment:
use `JSON.stringify(packageFile, null, '  ')`. That way it will be pretty-printed.